repo/blob/s3/s3_storage_test.go

```diff
@@ -11,7 +11,12 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Are there minio equivalent packages and structs for these?

Reply:
@julio-lopez we are leveraging `aws assume-role` to create temporary creds. For that we need to use aws sdks.